go-controller/go.mod

```diff
@@ -31,5 +31,6 @@ require (
 	k8s.io/apimachinery v0.17.4
 	k8s.io/client-go v0.17.4
 	k8s.io/klog v1.0.0
+	k8s.io/kubernetes v1.13.0
```

Comment:
I would strongly suggest against vendoring kubernetes. Copy the needed logic if you'd like.

Reply:
Ack.